Does not build in linux 6.13.0

I've tried building it through the AUR rtl8821ce-dkms-git package. 


> (2/5) Install DKMS modules
> ==> dkms install --no-depmod rtl8821ce/1.0.5.r172.gf119398 -k 6.6.72-1-lts
> ==> dkms install --no-depmod rtl8821ce/1.0.5.r172.gf119398 -k 6.13.0-2-cachyos
> 
> Error! Bad return status for module build on kernel: 6.13.0-2-cachyos (x86_64)
> Consult /var/lib/dkms/rtl8821ce/1.0.5.r172.gf119398/build/make.log for more information.
> ==> WARNING: `dkms install --no-depmod rtl8821ce/1.0.5.r172.gf119398 -k 6.13.0-2-cachyos' exited 10
> 

Full make.log is attached.

[make.log](https://github.com/user-attachments/files/18514109/make.log)

Same thing here

 ```
os_dep/linux/ioctl_cfg80211.c:9674:25: error: incompatible function pointer types initializing 'int (*)(struct wiphy *, struct net_device *, struct cfg80211_chan_def *)' with an expression of type 'int (struct wiphy *, struct cfg80211_chan_def *)' [-Wincompatible-function-pointer-types]
```

I've also builded it from the aur

--- a/os_dep/linux/ioctl_cfg80211.c
+++ b/os_dep/linux/ioctl_cfg80211.c
@@ -5868,6 +5868,9 @@ static int	cfg80211_rtw_set_channel(struct wiphy *wiphy
 }
 
 static int cfg80211_rtw_set_monitor_channel(struct wiphy *wiphy
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 13, 0)
+	, struct net_device *dev
+#endif
 #if (LINUX_VERSION_CODE >= KERNEL_VERSION(3, 8, 0))
 	, struct cfg80211_chan_def *chandef
 #else

